app/battle_preparation/battle_pref.py

```diff
@@ -0,0 +1,34 @@
+def battle_pref(knights: dict) -> dict:
+    new_pref_knights = {}
+
+    # loop for knight
+    for knight in knights:
+        each_kn = knights[knight]
+
+        each_kn["protection"] = 0
+
+        for armour_part in each_kn["armour"]:
+            each_kn["protection"] += armour_part["protection"]
+
+        # apply weapon
+        each_kn["power"] += each_kn["weapon"]["power"]
+
+        # apply potion if exist
+        if each_kn["potion"] is not None:
+            if "power" in each_kn["potion"]["effect"]:
+                each_kn["power"]\

```

Comment:
use braces instead of back slashes

Reply:
I changed my code, so now i have not E501 and no need to make slash or braces. But i did not understand, what did u mean, how i need do postponement with braces?